### DISCUSSION TITLE
Welcome to steady Discussions!

### DISCUSSION THREAD

**Discussion by Panthro** [Panthro/steady#2] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Panthro** [comment]
Just a comment

**Reply by Panthro** [reply]
> Just a comment

replying to see how it looks like

**Reply by Panthro** [reply]
This is a much nicer way to have replies!

**Reply by allousas** [reply]
> This is a much nicer way to have replies!

Replying to your reply

**Comment by Panthro** [comment]
Another comment

**Reply by Panthro** [reply]
I can also reply without a quote!
